sphinx/ext/intersphinx/_shared.py

```diff
@@ -4,11 +4,13 @@
 
 from typing import TYPE_CHECKING, Union
 
+from sphinx.util import logging
 from sphinx.util.typing import Inventory
 
 if TYPE_CHECKING:
     from sphinx.environment import BuildEnvironment
 
+LOGGER = logging.getLogger('sphinx.ext.intersphinx')
```

Comment:
(I'd suggest having a `Final` annotation for this one)

Reply:
done